src/workerd/api/crypto/crypto.c++

```diff
@@ -538,18 +542,17 @@ jsg::Promise<jsg::Ref<CryptoKey>> SubtleCrypto::unwrapKey(jsg::Lock& js,
 
     validateOperation(unwrappingKey, normalizedAlgorithm.name, CryptoKeyUsageSet::unwrapKey());
 
-    kj::Array<kj::byte> bytes =
-        unwrappingKey.impl->unwrapKey(kj::mv(normalizedAlgorithm), wrappedKey);
+    auto bytes = unwrappingKey.impl->unwrapKey(js, kj::mv(normalizedAlgorithm), wrappedKey);
```

Comment:
Not really a comment for this PR, but I wish we had less auto and more explicit types, and here you are going the other way.  For the reader it's just less clear.  Could be that the IDE tells you the type, but we don't always have that (eg here in the code review).

Reply:
Noted ;-) .... I think we've found the c++ equivalent to spaces vs. tabs tho lol ... some folks seem to really like `auto` (like myself) and others really hate it